tools/build_pak_files.py

```diff
@@ -42,12 +43,16 @@
 
     assert os.path.exists(shared_dir)
     unreal_project_content_shared_dir = os.path.join(unreal_project_content_dir, "Shared")
-    if not os.path.exists(unreal_project_content_shared_dir):
-        print(f"[SPEAR | build_pak_files.py] Creating symlink: {unreal_project_content_shared_dir} -> {shared_dir}")
-        os.symlink(shared_dir, unreal_project_content_shared_dir)
+
+    if spear.path_exists(unreal_project_content_shared_dir):
```

Comment:
Add these comments to match the comments inside the for loop, or remove the comments from inside the for loop

```
# remove existing shared dir
# create symlink
```